### ISSUE TITLE
How to set learning rate decay in lisa fine-tuning

### ISSUE THREAD

**Issue by orderer0001** [OptimalScale/LMFlow#775]
How to set learning rate decay in lisa fine-tuning

**Comment by research4pan** [comment]
Thanks for your interest in LMFlow! You may specify the learning rate schedule in the `./scripts/run_finetune_with_lisa.sh` as follows,

```
...
python3 example/finetune.py \
  --lr_scheduler_type cosine \
 ...
```

For supported schedule types, you may refer to this page (https://github.com/huggingface/transformers/blob/main/src/transformers/trainer_utils.py#L405). Hope this information can be helpful 😄

**Comment by orderer0001** [comment]
I get it, thank
